shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/main/antlr4/imports/Keyword.g4

```diff
@@ -78,3 +90,36 @@ NAME
 PROPERTIES
     : P R O P E R T I E S
     ;
+
+VARIABLE
+    : V A R I A B L E
+    ;
+
+HINT
+    : H I N T
+    ;
+
+SOURCE
+    : S O U R C E
+    ;
+    
+STATUS
+    : S T A T U S
+    ;
+
+SHARDING
```

Comment:
Do you think `sharding` and `read-write-splitting` are supposed to move to their own feature distsql packages?

Reply:
It's really better, thanks for the suggestion.